go.mod

```diff
@@ -8,6 +8,7 @@ require (
 	github.com/knadh/koanf v0.14.0
 	github.com/onsi/ginkgo v1.14.2
 	github.com/onsi/gomega v1.10.4
+	github.com/operator-framework/operator-lib v0.1.0
```

Comment:
I'm using the operator-lib here because the relevant Kubernetes implementation [is only available in 1.19+](https://github.com/kubernetes/apimachinery/blob/d4f471b82f0a17cda946aeba446770563f92114d/pkg/apis/meta/v1/types.go#L1368). I asked during one of the daily meetings what the minimal requirements for k8up are and I was told it's currently k8s 1.18. So I opted for the implementation provided by the operator-lib.

Since these types were remove in operator-framework 0.2.0 in favour of the upstream implementation this will surface eventually.

Reply:
Yes, I pointed that out already. However with K8s 1.19 we won't be needing operator-lib anymore just for the status, K8s comes with its own helper methods, as mentioned.
That's why i asked for a comment stating that it's temporary until we managed to upgrade to 1.19

Reply:
You'll now find an improved note in `api/v1alpha1/status.go`.